paymentsheet/src/test/java/com/stripe/android/link/ui/wallet/WalletScreenTest.kt

```diff
@@ -519,10 +548,9 @@ internal class WalletScreenTest {
         composeTestRule.waitForIdle()
 
         onWalletPaymentMethodMenu().assertDoesNotExist()
-        onWalletPaymentMethodRowMenuButton().onLast().performClick()
+        onWalletPaymentMethodRowMenuButton().onFirst().performClick()
```

Comment:
why was this change needed?

Reply:
I'm actually not sure. But given that we're testing for the existence of something (and not the non-existence), this should cover things equally well.